urbansearch/utils/dataset_p_utils.py

```diff
@@ -0,0 +1,148 @@
+import config
+import datetime
+import os
+import pickle
+
+from .p_utils import PickleUtils
+
+CATEGORIES = [
```

Comment:
Shouldn't we define the categories globally somewhere? In case new ones need to be added, we will now need to adjust the code in multiple places.. @pvanagtmaal @TomBrunner